gdcdictionary/schemas/follow_up.yaml

```diff
@@ -1321,15 +1525,24 @@ properties:
         $ref:
           - "_terms_enum.yaml#/not_reported/common"
 
+  evidence_of_progression_type:
+    $ref:
+      - "_terms.yaml#/evidence_of_progression_type/common"
```

Comment:
I don't think there is a corresponding entry in `_terms`.yaml for `evidence_of_progression_type` which may be the reason the build is failing

Reply:
@wwysoc2 and @mpsolano I'll update this.

Reply:
👍